packages/jest-each/src/table/interpolation.ts

```diff
@@ -77,7 +77,11 @@ export function getPath(
   template: Template,
   [head, ...tail]: Array<string>,
 ): unknown {
-  if (!head || !template.hasOwnProperty || !template.hasOwnProperty(head))
+  if (
+    !head ||
+    !template.hasOwnProperty ||
```

Comment:
do we need this one?

Reply:
What's your bet? Would like to know your POV

Reply:
I don't understand why we need this check when we're not calling the function

Reply:
@SimenB, yeah totally agree in that i guess we dont need this one